thorlcr/graph/thgraphmaster.cpp

```diff
@@ -688,25 +701,21 @@ void CMasterActivity::updateFileReadCostStats()
     {
         IDistributedFile *file = queryReadFile(0);
         if (file)
-        {
-            stat_type numDiskReads = statsCollection.getStatisticSum(StNumDiskReads);
-            StringBuffer clusterName;
-            file->getClusterName(0, clusterName);
-            diskAccessCost += money2cost_type(calcFileAccessCost(clusterName, 0, numDiskReads));
-            file->addAttrValue("@numDiskReads", numDiskReads);
-        }
+            diskAccessCost += updateReadCosts(file, statsCollection);
     }
 }
 
 void CMasterActivity::updateFileWriteCostStats(IFileDescriptor & fileDesc, IPropertyTree &props, stat_type numDiskWrites)
 {
     if (numDiskWrites)
     {
-        props.setPropInt64("@numDiskWrites", numDiskWrites);
+        props.setPropInt64(getDFUQResultFieldName(DFUQRFnumDiskWrites), numDiskWrites);
         assertex(fileDesc.numClusters()>=1);
         StringBuffer clusterName;
         fileDesc.getClusterGroupName(0, clusterName);// Note: calculating for 1st cluster. (Future: calc for >1 clusters)
-        diskAccessCost = money2cost_type(calcFileAccessCost(clusterName, numDiskWrites, 0));
+        cost_type writeCost = money2cost_type(calcFileAccessCost(clusterName, numDiskWrites, 0));
+        props.setPropInt64(getDFUQResultFieldName(DFUQRFwriteCost), writeCost);
+        diskAccessCost += writeCost;
```

Comment:
I think this will over-accumulate the costs.  Should be 
diskAccessCost = writeCost
otherwise each time it is called all previous costs will be added again.